src/app/quiz/body.js

```diff
@@ -0,0 +1,67 @@
+'use client'
+
+import Box from "@mui/material/Box";
+import {useTheme} from '@mui/material/styles';
+import Container from "@mui/material/Container";
+import Grid from '@mui/material/Grid';
+
+function VariantButton() {
+  const theme = useTheme()
+  return (
+    <Box
+      sx={{
+        backgroundColor: theme.palette.quiz.button.backgroundColor,
+        p: 3,
+        borderRadius: 0.6,
+        margin: 0.6,
```

Comment:
Do not try to make spacing inside of a grid by element padding. There are columnSpacing and rowSpacing of Grid container itself.